src/db/addSectionMgmt.sql

```diff
@@ -392,15 +279,14 @@ RETURNS TABLE(Number VARCHAR(8),
             Difference INTEGER)
 AS
 $$
-
-SELECT CASE 
-   WHEN c.title = $2
-   THEN number, title, 0
-   ELSE number, title, 1
+SELECT number, title, 
+   CASE WHEN title = $2
+      THEN 0
+      ELSE 1
+   END
```

Comment:
This looks a lot better; thanks for finding out that this works too.

Reply:
Yes, had some issues testing it out, and this was the result.  In either case, it's a bit easier to read.